src/pages/Components/Tags/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: 'Tags'
+order: 6
+---
+
+> Tags are used to categorize, show filters or make selections. They are compact components mostly used as secondary content when you have to deal with a product property.
+
+-- Preview of the four tags --
+
+## Four Tags variations for different usages
+
+### Text tags
+
+Clickable tags are used to display relative keywords of an item or a content. They are used to create a cloud tag when a content live in a multiple categories.
+
+-- Preview text tags --
+
+### Link tags
+
+Link tags are a variation of the text tags, they have the same appearance but they are **used as a link**.
+
+-- Preview text tags --
+
+### Selectable tags
+
+Selectable tags are used to select (or deselect) values from a proposal of multiple keywords. They are a good alternative to radio buttons, checkboxes or toggle buttons when you need to select values **outside a form**.
+
+-- Preview selectable tags --
+
+### Removable tags
+
+Removable tags are indicators for the users, they are the result of the previously selected values and can be removed by taping.
```

Comment:
Removable tags are interactive indicators of a previous selection that can be removed by taping/clicking on a remove button.